green-hub/src/components/GetPointsBox/GetPointsBox.module.css

```diff
@@ -229,6 +229,7 @@ input::-webkit-inner-spin-button {
 
 @media (max-width: 520px) {
     .getpointsBox {
+        min-width: 93%;
```

Comment:
egzotyczny wymiar :)

Reply:
to nie jest przypadkowa liczba, tak pasowało :P